src/styles/style.css

```diff
@@ -0,0 +1,77 @@
+* {
```

Comment:
Discussed to maintain the CSS styles at the component level instead of one big css file.